### ISSUE TITLE
Update Design System Components to Adhere to WCAG 2.2 Standards

### ISSUE THREAD

**Issue by JoeTice** [department-of-veterans-affairs/va.gov-team#37333]
## Issue Description
Based on assessments and the discussions with the Design System & Governance Teams

---
## Tasks
- [ ] Create subtasks based on the findings from the assessments & discussions with the Design System & Governance team - tickets should reflect the action items as needed
- [ ] Complete task tickets

## Acceptance Criteria
- [ ] All stakeholder groups verify tasks have been completed and documentation is complete

**Comment by JoeTice** [comment]
Format for tickets should be...

## Issue Description
WRITE SPECIFIC UPDATE DETAILS HERE

---
## Tasks
- [ ] Update component to support new standard
- [ ] Update tests to account for changes & new standards
- [ ] Update component documentation

## Acceptance Criteria
- [ ] Update is made and works as intended
- [ ] Tests work properly and detect no defects
- [ ] Documentation has been reviewed and approved
